graphql_api/types/owner/owner.py

```diff
@@ -81,6 +82,7 @@ def resolve_plan_representation(owner: Owner, info) -> PlanData:
 
 @owner_bindable.field("availablePlans")
 @convert_kwargs_to_snake_case
+@require_part_of_org
```

Comment:
what was the determining factor in picking which functions to add the decorator to? I see there are certain functions where we didn't add it so was just curious

Reply:
https://www.notion.so/sentry/GraphQL-Owner-Type-Sensitive-Fields-f78cb53e77e4428aa6e04b2819d67884
and yes, good catch, added in all the fields